openlibrary/templates/site/banner.html

```diff
@@ -8,7 +8,7 @@
 $# cookie_name (str) -- identifier for the banner. If the user dismisses the banner, a cookie with this name is set
 $# cookie_duration_days (int) -- TTL before cookie expires in days
 
-$ hide_banner = ctx.user and ctx.user.has_cookie(cookie_name)
+$ hide_banner = cookies().get(cookie_name)
```

Comment:
This change and https://github.com/internetarchive/openlibrary/pull/9432/files#diff-36bcbe01eb6ced7fe08b08f87ee1c4c4e70c4e41cbbaabd89b08de62ad6f8cdfR1123 should both land independently of this PR as they presumably fix a long-standing issue with dismissible banners

Reply:
Resolved by #9436